lib/loader/JSMockLoader.js

```diff
@@ -27,9 +27,16 @@ var JSMock = require('../resource/JSMock');
 function JSMockLoader(options) {
   ResourceLoader.call(this, options);
 
+  var extensions;
+  if(options && options.extensions){
+    extensions = options.extensions;
+  }else{
```

Comment:
Style nit:
`if (options && options.extensions) {` (space before opening paren, before opening brace, ditto for the else below)